orttraining/orttraining/core/session/training_session.cc

```diff
@@ -289,14 +354,28 @@ Status TrainingSession::ConfigureForTraining(
       weight_names_to_train, loss_name, config.gradient_graph_config, *session_logger_));
 
   if (config.pipeline_config.has_value()) {
+    // TODO: The number of pipeline steps should be passed in from Python API.
```

Comment:
Is this piece now done -- the 0.5 example is passing in trainer_config.distributed.num_pipeline_steps

Reply:
Removed.

---
In reply to: [527649238](https://github.com/microsoft/onnxruntime/pull/5815#discussion_r527649238) [](ancestors = 527649238)